src/screens/Login/Login.jsx

```diff
@@ -1,5 +1,56 @@
+import "./login.css";
+import { Link } from "react-router-dom";
+
 const Login = () => {
-  return <>Login</>;
+  return (
+    <main className="login flex_col flex_align__center mb_10 pt_8">
+      <h1 className="h2 ">Login</h1>
+      <form className="flex_col my_8 mx_auto">
+        <label className="label field_required" htmlFor="email_id">
+          Email address
+        </label>
+        <input
+          id="email_id"
+          type="email"
+          placeholder="sheldon@tbbt.com"
+          className="input block"
+          required
+        />
+        <div className="login_pwd">
+          <label htmlFor="password" className="label field_required" required>
+            Password
+          </label>
+          <input
+            type="password"
+            id="password"
+            className="input block"
+            placeholder="Your password here..."
+            required
+          />
+          <a className="btn_icon btn_show__password">
```

Comment:
No need of anchor tag here

Reply:
Thanks for pointing that out, fixed in https://github.com/saadahmadhere/Wisdom-library/pull/4/commits/023da4d2dc191103e163c209ab1b3ee261d9317c